src/adapters/suggest.js

```diff
@@ -95,23 +95,14 @@ export default class Suggest {
 
       onSelect: (e, term, item, items) => {
         e.preventDefault()
-        const type = item.getAttribute('data-type')
-        const val = item.getAttribute('data-val')
         const itemId = item.getAttribute('data-id')
 
-        if ('category' === type) {
-          PanelManager.openCategory({
-            category: CategoryService.getCategoryByName(val)
-          })
-        }
-        else {
-          let prefixPoint = this.prefixes.find((prefix) => prefix.id === itemId)
-          if(prefixPoint) {
-            this.onSelect(prefixPoint)
-          } else {
-            let poi = items.find(poi => poi.id === itemId)
-            this.onSelect(poi)
-          }
+        let prefixPoint = this.prefixes.find((prefix) => prefix.id === itemId)
```

Comment:
Just an idea: comparisons in JS are faster when they are strict. Why not using `findIndex` and then compare like this: `if (prefixPoint !== -1) {`?

Reply:
I made a change to use `if (prefixPoint !== undefined)`  
This function being called a few times per session, performance is probably not significant here.